doc/rbd/iscsi-requirements.rst

```diff
@@ -0,0 +1,53 @@
+==========================
+iSCSI Gateway Requirements
+==========================
+
+To implement the Ceph iSCSI gateway there are a few requirements. It is recommended
+to use two to four iSCSI gateway nodes for a highly available Ceph iSCSI gateway
+solution.
+
+For hardware recommendations, see the `Hardware Recommendation page <http://docs.ceph.com/docs/master/start/hardware-recommendations/>`_
+for more details.
+
+.. WARNING::
+    On the iSCSI gateway nodes, the memory footprint of the RBD images
+    can grow to a large size. Plan memory requirements accordingly based
+    off the number RBD images mapped. Each RBD image roughly uses 90 MB
+    of RAM.
+
+There are no specific iSCSI gateway options for the Ceph Monitors or
+OSDs, but changing the ``osd_client_watch_timeout`` value to ``15`` is
+required for each OSD node in the storage cluster.
+
```

Comment:
We can drop the osd_client_watch_timeout info.

We need to set these on the OSDs:

[osd]
osd heartbeat grace 20
osd heartbeat interval 5

There should be a note that for windows/linux these settings being shorter than the initiator command/Disk timeouts used in the initiator part of the guide.

We cannot configure the disk/command timeout in ESX, so the settings can be used.